blerry/blerry.be

```diff
@@ -87,6 +88,12 @@ class blerry_helpers
     f.close()
   end
 
+  static def write_url(url)
+    var f = open("blerry_configurl.txt", "w")
```

Comment:
Looks like you're passing the filename as a variable some places and hardcoded other places

Reply:
Looks like we are hardcoding things in a lot of places. Removed the need to pass the filename, and hardcoded it.